agent/ansible/ara/pbench-ara.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: all
+  user: root
+  roles:
+  - role: pbench-ara-role
```

Comment:
We have an rpm - https://copr.fedorainfracloud.org/coprs/ndokos/pbench/package/pbench-ara-role/ which places the pbench-ara-role in /etc/ansible/roles.